device-types/Supermicro/CloudDC-SuperServer-SYS-620C-TN12R.yaml

```diff
@@ -0,0 +1,23 @@
+---
+manufacturer: Supermicro
+model: CloudDC SuperServer SYS-620C-TN12R
+slug: supermicro-sys-620c-tn12r
+part_number: SYS-620C-TN12R
+u_height: 2
+is_full_depth: true
+weight: 31.3
+weight_unit: kg
+airflow: front-to-rear
+comments: 'Specification: [https://www.supermicro.com/en/products/system/cloud/2u/sys-620c-tn12r](https://www.supermicro.com/en/products/system/cloud/2u/sys-620c-tn12r)'
+power-ports:
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 1200
+  - name: PSU2
+    type: iec-60320-c14
+    maximum_draw: 1200
```

Comment:
These PSUs appear to be modular. If that is true, these should be defined as `module-bays` and the associated PSUs should be created in the module-types folder